lib/rubocop/cop/rails/save_bang.rb

```diff
@@ -0,0 +1,75 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop identifies possible cases where Active Record save! or related
+      # should be used instead of save because the model might have failed to
+      # save and an exception is better than unhandled failure.
+      #
+      # This will ignore calls that are assigned to a variable or used as the
+      # condition in an if/unless statement.  It will also ignore any call with
+      # more than 2 arguments as that is likely not an Active Record call or
+      # if a Model.update(id, attributes) call
+      #
+      # @example
+      #
+      #   # bad
+      #   user.save
+      #   user.update(name: 'Joe')
+      #   user.find_or_create_by(name: 'Joe')
+      #   user.destroy
+      #
+      #   # good
+      #   unless user.save
+      #      . . .
+      #   end
+      #   user.save!
+      #   user.update!(name: 'Joe')
+      #   user.find_or_create_by!(name: 'Joe')
+      #   user.destroy!
+      class SaveBang < Cop
+        MSG = 'Use %s instead of %s if the return value is not checked.'.freeze
+
+        SAVE_METHODS = [:save, :create, :update, :destroy,
```

Comment:
Is there a general term for interacting with the database other than "save"? Maybe "persist".